packages/easy-hydrator/src/ParameterValueGetter/ParameterValueGetterInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php declare(strict_types=1);
+
+namespace Symplify\EasyHydrator\ParameterValueGetter;
+
+use ReflectionParameter;
+
+interface ParameterValueGetterInterface
+{
+    /**
+     * @param mixed[] $data
+     * @return mixed
+     */
+    public function getValue(ReflectionParameter $reflectionParameter, array $data);
+}
```

Comment:
This interface can be dropped, it's never used. Probably forgotten

Reply:
Ah, not forgotten but missed after fucked up rebase 🙄 😄 thx for this catch